src/Swarm/TUI/Model.hs

```diff
@@ -1053,3 +1054,4 @@ scenarioToUIState siPair u =
       & uiREPL .~ initREPLState (u ^. uiREPL . replHistory)
       & uiREPL . replHistory %~ restartREPLHistory
       & scenarioRef ?~ siPair
+      & lastFrameTime .~ curTime
```

Comment:
This makes me think we should have something like a `UIGameState` that is reset on a new game. :thinking: 

It should not be possible to carry over things between games both in `GameState` and in `UIState`.

Reply:
Or perhaps it means `lastFrameTime` really ought to live in `GameState` (once we fix #516 and #689).